tests/neo-vm.Tests/Extensions/ByteArrayExtensions.cs

```diff
@@ -20,10 +20,5 @@ public static byte[] Sha256(this IEnumerable<byte> value)
         {
             return _sha256.Value.ComputeHash(value.ToArray());
         }
-
```

Comment:
Nice job as always, @shargon.

I also show that you cleaned duplicated code, right?

Reply:
Yes, this method is not used